src/utils/convert3Dto4D.m

```diff
@@ -0,0 +1,192 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function convert3Dto4D
```

Comment:
I suspect this is more an example of how to use it, no?

So it might be better in the `demo` folder.

Reply:
now there is a demo with a batch to run this code